### PR TITLE
feat(Wolvesville): more statuses, improvements

### PR DESCRIPTION
**Pull request by down-bad**
Fixed bugs from new game updates and from having the game in light mode (gotta love react native web), improved and added statues, some refactoring.

### CODE REVIEW

File: websites/W/Wolvesville/dist/metadata.json [hunk 1]
```diff
@@ -104,6 +104,7 @@
 			"value": 0,
 			"values": [
 				"Normal",
+        "White",
```
Comment (EncryptedDev) [inline websites/W/Wolvesville/dist/metadata.json:107]: Unformatted or broken diff?

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/W/Wolvesville/dist/metadata.json (hunk 1 above)
Comment (down-bad) [reply websites/W/Wolvesville/dist/metadata.json:107]: it had spaces instead of tabs, fixed

**Comment by down-bad** [comment]
Unfortunately no matter the ~2 weeks of testing, I'll have to fix something tomorrow cuz React is full of surprises, in case u were planning on merging. ty 👍

**Comment by down-bad** [comment]
all done

**Review by EncryptedDev — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**